app/Http/Controllers/FAQController.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace App\Http\Controllers;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\DB;
+use function Laravel\Prompts\alert;
+
+class FAQController extends controller
+{
+    public function createfaqpost(Request $request)
```

Comment:
A naming issue here: because this is a method in the FAQController class, adding 'faq' in this method name is redundant. Just `create` might just do fine here.